src/components/DashboardAnalytics/ProfitSharingAnalytics/ProfitSharingAnalytics.js

```diff
@@ -184,7 +184,10 @@ const ProfitSharingAnalytics = ({ provider }) => {
       ) : (
         balanceHistory && (
           <>
-            <TradingPerformance performance={stats.performance} unit=" USDT" />
+            <TradingPerformance
+              performance={stats.performance}
+              unit={balanceHistory && balanceHistory.quote ? balanceHistory.quote : ""}
```

Comment:
You can just to `balanceHistory?.quote`
or `balanceHistory?.quote || ''` if needs to be empty string by default

Reply:
well balanceHistory is null initially thats why i added this.

Reply:
Yeah but the syntax I gave you takes care of that: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
it does? okay changing it

Reply:
Done.